dist/icon/dashboard/order.svg

```diff
@@ -1 +1 @@
-<svg viewBox="0 0 12 12" xmlns="http://www.w3.org/2000/svg" id="ds-icon-dashboard-order"><path id="direction-dsc" d="m1.122 7.624 2.032 2.318a.478.478 0 0 0 .358.169c.134 0 .263-.06.358-.169l2.032-2.318a.632.632 0 0 0 .121-.4.62.62 0 0 0-.148-.387.48.48 0 0 0-.34-.169.47.47 0 0 0-.35.139L4.02 8.134V2.58a.623.623 0 0 0-.149-.41.478.478 0 0 0-.36-.17.478.478 0 0 0-.358.17.623.623 0 0 0-.149.41v5.554L1.838 6.807a.47.47 0 0 0-.35-.139.48.48 0 0 0-.34.17.62.62 0 0 0-.148.387.632.632 0 0 0 .122.4Z"/><path id="direction-asc" d="M6.098 4.487 8.13 2.169A.478.478 0 0 1 8.488 2c.134 0 .263.06.358.169l2.032 2.318c.083.11.127.253.122.399a.622.622 0 0 1-.149.387.481.481 0 0 1-.34.17.47.47 0 0 1-.35-.14L8.997 3.978V9.53a.623.623 0 0 1-.149.41.478.478 0 0 1-.359.17.478.478 0 0 1-.359-.17.623.623 0 0 1-.149-.41V3.977L6.814 5.304a.47.47 0 0 1-.35.139.48.48 0 0 1-.339-.17.62.62 0 0 1-.148-.387.632.632 0 0 1 .121-.4Z"/></svg>
```

Comment:
Seems like something changed that's removing the IDs. That'll break our icon CSS rules.

Reply:
Good catch, this seems to change every other time we run this 😅